test/sqllogictest/autogenerated/mz_internal.slt

```diff
@@ -559,6 +559,7 @@ mz_dataflow_operators
 mz_dataflow_operators_per_worker
 mz_dataflows
 mz_dataflows_per_worker
+mz_expected_group_size_advice
```

Comment:
You might have to update `mz_internal.md` to keep this in sync. I'm not sure why the lint didn't run.

Reply:
I think that there is a [general preference for separating out the docs PR](https://materializeinc.slack.com/archives/CPNFV9CVB/p1691665154023779). So the documentation item is missing in this PR, because I was going to submit it separately. Do we need to do it together (not sure how the linter works)?

Reply:
Yup, oversight in the CI pipeline: https://github.com/MaterializeInc/materialize/pull/21106

Reply:
If you rebase now, it should trigger the right set of lints.

Reply:
Cool, rebased and provided an initial documentation for the view. I will make sure to update the description later when I submit a docs PR.